Source/i_sound.c

```diff
@@ -678,13 +684,7 @@ void I_ShutdownSound(void)
 //
 void I_InitSound(void)
 {   
-   // [FG] initialize music backend function pointers
-   if (music_backend == music_backend_opl)
-      I_OPL_InitMusicBackend();
-   else
-      I_SDL_InitMusicBackend();
-
-   if(!nosfxparm)
+   if(!nosfxparm || !nomusicparm)
```

Comment:
> Yes, let's please fix the `-nosfx` parameter. It is strictly supposed to work independent of the music backend.

I fixed it with this change. Not sure if it's worth adding SDL initialization to every music module like in Choco, I think it's not necessary.